api/envoy/type/matcher/v3/regex.proto

```diff
@@ -62,7 +62,8 @@ message RegexMatcher {
     GoogleRE2 google_re2 = 1 [(validate.rules).message = {required: true}];
   }
 
-  // The regex match string. The string must be supported by the configured engine.
+  // The regex match string. The string must be supported by the configured engine. The regex should
```

Comment:
I'm not thrilled with this wording. Anyone have a better way to phrase it?

Reply:
`The string supported by the specified regex engine`?

Reply:
How about "The regex is matched against the full string, not as a partial match."